### ISSUE TITLE
Verify that MecDriver works using omni-wheels.

### ISSUE THREAD

**Issue by AhmedSamara** [IEEERobotics/bot#179]
At the last hack-day @LizardChen pointed out a very interesting property of the omni wheels.

If they're placed at the corners like so:

![](http://www.robot-hk.com/products/600/KWR-0111-001a.png)

They behave exactly like mecanum wheels. They all push off at 45 degree angles, which should mean that the vectors can be played against each other to vary the translation angle.

Even if you think about it without doing any math, you can see that if you wanted to move diagonally, you would only turn on the front and back motors of opposite sides and leave the others off, which is exactly what the mecanum wheels do just because of how the trigonometry in mecdriver works out.

Even if we place them on the sides (still orthogonal to each other), this will still work out, just with one of the corners being the 'front' of the bot.

In other words, if what @LizardChen pointed out is right, MecDriver can pretty much be used as is with the Omni Wheel bot, but also with some of the other benefits of omni-wheels.

**Comment by AhmedSamara** [comment]
This is worth looking into again.
